### PR TITLE
Upload tags

### LINKED ISSUE #122: Allow assigning tags at upload time
So, for example, you can upload a known benign file and tag it as such in one step.

### PR DESCRIPTION
**Pull request by clenk**
Fixes #122.

### CODE REVIEW

**Comment by ptcNOP** [comment]
Does this support the tag autocomplete for defined tags like on the results page?

**Comment by clenk** [comment]
It should now.

**Comment by ptcNOP** [comment]
Not sure that this actually matters, but any idea why the default value [here](https://github.com/clenk/multiscanner/blob/851c243be54c9edadd2025f2c07e7031d23fb51d/web/app.py#L85) is an empty dict and [here](https://github.com/clenk/multiscanner/blob/851c243be54c9edadd2025f2c07e7031d23fb51d/web/app.py#L67) it's an empty list?

File: web/templates/index.html [hunk 1]
```diff
@@ -218,7 +218,7 @@
     // Set up tags
     $.get("{{ api_loc }}/api/v1/tags/", function(data, status) {
       $.each(data.tags, function( index, value ) {
```
Comment (ptcNOP) [inline web/templates/index.html]: This endpoint returns tags in the `Tags` key (notice capital `T`).

Comment (clenk) [reply web/templates/index.html]: Good catch!

**Comment by clenk** [comment]
It should default to a list, not a dict.

**Pull request merged**